src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -158,6 +143,14 @@ public function setupFieldAssociations($fields, $table, $fieldConditions, $strat
                 $fieldTable = TableRegistry::get($name);
             }
 
+            $conditions = [
+                $name . '.model' => $model,
+                $name . '.field' => $field,
+            ];
+            if (!$this->_config['allowEmptyTranslations']) {
```

Comment:
There are (almost) no other uses of `config()` in the behavior - just following suit. Also, there's no possibility of these keys being missing.